### DISCUSSION TITLE
npm run pro-> class extends Sv.Component{constructor(){super()}componentMount(){}

### DISCUSSION THREAD

**Discussion by skygst102** [webpack/webpack#16711] [Q&A]
# Bug report

**What is the current behavior?**
npm run pro
class extends Sv.Component{constructor(){super()}componentMount(){}
**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
npm run pro
class c1 extends Sv.Component{constructor(){super()}componentMount(){}

**Other relevant information:**
webpack version:  5.75
Node.js version: 14.18.0
Operating System: win10
Additional tools:

**Comment by alexander-akait** [comment]
Please create reproducible test repo
